src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -680,15 +686,17 @@ def __call__(self):
 
         try:
             origCores = int(getattr(self.step.data.application.multicore, 'numberOfCores', 1))
+            eventStreams = int(getattr(self.step.data.application.multicore, 'eventStreams', 0))
             resources = {'cores': origCores}
-            resizeResources(resources)
+            if not eventStreams:  # Don't know how to handle resizing jobs with different nThreads and nStreams
```

Comment:
OK, you are bringing me around. I will change the logic. I'll also put something in the log file.